packages/esm-commons-lib/package.json

```diff
@@ -39,8 +39,8 @@
     "yup": "^0.29.1"
   },
   "peerDependencies": {
-    "@openmrs/esm-framework": "5.x",
-    "@openmrs/esm-patient-common-lib": "6.x",
+    "@openmrs/esm-framework": "*",
+    "@openmrs/esm-patient-common-lib": "*",
```

Comment:
These should be X.x versions, right @ibacher?

Reply:
Yes, although I don't know that that's a hard requirement.